playbooks/tasks/logging/configure_logging_agent.yml

```diff
@@ -0,0 +1,181 @@
+---
+- name: Configure Logging Agent
+  hosts: hetzner
+  gather_facts: false
+  vars_prompt:
+    - name: user
+      prompt: Enter the Opensearch User
+    - name: password
+      prompt: Enter the Opensearch Password
```

Comment:
Playbooks should be runnable without prompts. Can you find another way to distribute secrets? So far I've used a .gitignored `secrets.yml` group_vars file